### ISSUE TITLE
Blacklight8: Selected facet has insufficient contrast with background

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/orangelight#4530]
### Expected behavior
Selected facets are accessible and have sufficient contrast

### Actual behavior
Selected facets have insufficient contrast

Blacklight 7 (passes contrast check)
![image](https://github.com/user-attachments/assets/80b7d2ad-afee-434d-8a28-6fe4e06a780e)

Blacklight 8 (fails contrast check)
![image](https://github.com/user-attachments/assets/4a80cdb4-e4f5-4923-86b0-95c57d88c1dd)

### Steps to replicate
Either:
a. Go to https://catalog-qa.princeton.edu/?f%5Blocation%5D%5B%5D=Mudd+Manuscript+Library
b. Go to library home page and select a facet to filter by

### Impact of this bug
Not accessible, can't upgrade to Blacklight 8
